yarn.lock

```diff
@@ -3090,7 +3090,7 @@
   resolved "https://registry.yarnpkg.com/@tootallnate/once/-/once-2.0.0.tgz#f544a148d3ab35801c1f633a7441fd87c2e484bf"
   integrity sha512-XCuKFP5PS55gnMVu3dty8KPatLqUoy/ZYzDzAGCQ8JNFCkLXzmI7vNHCR+XpbZaMWQK/vQubr7PkYq8g470J/A==
 
-"@ts-morph/bootstrap@0.19.0":
+"@ts-morph/bootstrap@0.18.0":
```

Comment:
Interesting

Reply:
Hmmm got autogenerated but I can revert if needed. However, looking at the content, change seems reasonable :)

Reply:
Yeah I mean, if all tests pass then this should be moot.